Tasks/AzureFileCopyV2/task.json

```diff
@@ -14,7 +14,7 @@
   "version": {
     "Major": 2,
     "Minor": 230,
-    "Patch": 0
+    "Patch": 1
```

Comment:
Maybe need to bump the patch to 2 since 2.230.1 version already exists in production